### PR TITLE
Add links to tour on zerostate and deploy views

### PR DESCRIPTION
**Pull request by maciaszczykm**
Implements #476.

image

image

[Review on Reviewable](https://reviewable.io/reviews/kubernetes/dashboard/497)

### CODE REVIEW

**Comment by maciaszczykm** [comment]
Updated. PTAL

**Comment by maciaszczykm** [comment]
Updated. I hope this time it will be __the__ end of my mistakes :) PTAL

File: src/app/frontend/deploy/deploy.html [hunk 1]
```diff
@@ -28,6 +28,12 @@ <h3 class="md-headline kd-deploy-form-title">Deploy a Containerized App</h3>
             Upload a YAML or JSON file
           </md-radio-button>
         </md-radio-group>
+        <kd-user-help>
+          To learn more,
+          <a href="http://kubernetes.io/v1.1/docs/user-guide/ui.html" target="_blank">
```
Comment (bryk) [inline src/app/frontend/deploy/deploy.html]: Hmm.... The link here. It shows the old UI. See this announcement: https://groups.google.com/forum/#!topic/kubernetes-dev/UedwIYb-fHg

I think the correct link for 1.2 will be: http://kubernetes.github.io/docs/user-guide/ui/

What do you think?

Comment (maciaszczykm) [inline src/app/frontend/deploy/deploy.html]: I agree.

Comment (bryk) [inline src/app/frontend/deploy/deploy.html]: Ok, please update.

**Comment by maciaszczykm** [comment]
Updated. PTAL

**Comment by bryk** [comment]
LGTM. Will do final checs

**Pull request merged**